boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -91,8 +91,7 @@ gogenerate:
 
 .PHONY: opgenerate
```

Comment:
I'm just now seeing these Make targets. We seem to be very inconsistent in our naming convention here. Can we standardize on `<noun>-<verb>` in kebab-case? This would be consistent with `docker-build` and other targets. It would also make it more readable.
This would become `op-generate`...

Reply:
+1. Do you want to iterate on those names here or in the [card](https://issues.redhat.com/browse/OSD-4673)?

Reply:
Sorry, I just twigged that changing these target names isn't relevant to this PR at all, and only partly overlapping with the card referenced above (and its PR, #49).

Still +1 on renaming targets to follow some kind of pattern. But it needs to be done in a separate PR so we can keep this one focussed. Please do comment on [OSD-4673](https://issues.redhat.com/browse/OSD-4673) or #49 to refine the names of the targets to be used by prow and appsre; and propose any other renames separately.

Reply:
Fair enough, I commented on #49 to hopefully clean it up there. Thanks.